pkg/networkmanager/interfaces.js

```diff
@@ -3401,8 +3401,13 @@ PageNetworkIpSettings.prototype = {
                     });
         }
 
-        with_settings_checkpoint(PageNetworkIpSettings.model, modify,
-                                 { devices: connection_devices(PageNetworkIpSettings.connection) });
+        with_settings_checkpoint(
+            PageNetworkIpSettings.model,
+            modify,
+            {
+                devices: connection_devices(PageNetworkIpSettings.connection),
+                hack_does_add_or_remove: true
```

Comment:
The reason we have `hack_does_add_or_remove` is to avoid running into bugs in the NetworkManager checkpoint implementation: 

https://bugzilla.redhat.com/show_bug.cgi?id=1378393
https://bugzilla.redhat.com/show_bug.cgi?id=1398316

If there is a fundamental reason why we shouldn't use checkpoints in this case here, we should remove the call to `with_settings_checkpoint` entirely.

However, I think using checkpoints here is completely fine.

Reply:
> 
> 
> > If desired, I could also add another button in the IP Settings.
> 
> We have this dialog:
> 
> ![image](https://user-images.githubusercontent.com/3228183/83239427-87003b00-a1a0-11ea-894a-8e90f11a7534.png)
> 
> If this doesn't work for you, let's try to find the reason and fix that.

I also find the generally usage of checkpoints very useful. The checkpoint problem is only reproducible with the DSA device. In this case the dialog doesn't appear and a "Keep Connection" is not possible.
Only this message appears:
![image](https://user-images.githubusercontent.com/52933406/83497055-33e8f980-a4ba-11ea-89f8-54a3be9be9f8.png)
Is it possible to suppress the "Disconnected" event?